how-to/rpi-camera.rst

```diff
@@ -144,10 +145,37 @@ You can follow it to learn how to use ``picamera2`` on Ubuntu.
 
 .. important::
     Chapter 9.8 and 9.9 of the ``picamera2`` manual linked above make use of the Hailo and IMX500 AI accelerator. The Hailo
-    AI accelerator is not supported on Ubuntu, and the IMX500 AI accelerator requires the imx500-firmware package, which
+    AI accelerator is not supported on Ubuntu, and the IMX500 AI accelerator requires the ``imx500-firmware`` package, which
     can't be included in Ubuntu due to licensing issues. So anything in those chapters might not work out of the box.
     We are working on fixing the licensing issues with the involved parties.
 
+Workaround for the IMX500 (AI Camera)
+=====================================
+
+.. important::
+    While the steps given below will allow you to use the IMX500 camera module on Ubuntu,  the ``imx500-firmware``
+    package is not maintained in Ubuntu (yet).
+
+While ``imx500-firmware`` is not available in Ubuntu, you can still get it's deb from Raspberry Pi's `Debian pool`_.
+
+.. terminal::
+    :input: wget https://archive.raspberrypi.org/debian/pool/main/i/imx500-firmware/imx500-firmware_0.FF17.8_all.deb && sudo dpkg -i imx500-firmware_0.FF17.8_all.deb
```

Comment:
In the terminal examples, we should probably stick to user "ubuntu" and host "ubuntu" as those are the defaults on our images. It's a nit-pick, but it avoids anyone asking if specific users / groups / permissions are required for this (which they shouldn't be).